hw/ip/prim/util/primgen/abstract_prim.sv.tpl

```diff
@@ -15,7 +15,7 @@
 // abstract prim wrapper. These warnings occur due to the .*
 // use. TODO: we may want to move these inline waivers
 // into a separate, generated waiver file for consistency.
-//ri lint_check_off OUTPUT_NOT_DRIVEN INPUT_NOT_READ
+//ri lint_check_off OUTPUT_NOT_DRIVEN INPUT_NOT_READ HIER_BRANCH_NOT_READ
```

Comment:
You have to modify the corresponding `//ri lint_check_on` comment below as well - otherwise the tool gets upset and complains ;).

Reply:
Ah, that's a stupid mistake, sorry. It's now fixed.